data/lua/wml/harm_unit.lua

```diff
@@ -160,6 +160,10 @@ function wml_actions.harm_unit(cfg)
 
 			wesnoth.interface.float_label( unit_to_harm.x, unit_to_harm.y, string.format( "<span foreground='red'>%s</span>", text ) )
 
+			local award_kill_xp = experience == "kill" or experience == true
+			local award_primary_hit_xp = experience == "hit" or experience == "primary_hit" or experience == true
+			local award_secondary_hit_xp = experience == "hit" or experience == "secondary_hit" or experience == true
```

Comment:
Does not seem to support experience=kill,secondary_hit

Reply:
Yeah I have no idea how to do that yet, gotta wait until I learn I guess.